daemon/daemon_unix.go

```diff
@@ -16,7 +16,7 @@ import (
 	"strings"
 	"time"
 
-	containerd_cgroups "github.com/containerd/cgroups"
+	v1 "github.com/containerd/cgroups/stats/v1"
```

Comment:
this is kinda weird to call this `v1`. maybe just `cgroups`?

Reply:
"statsv1" (v2 will be added soon)

Reply:
Opened https://github.com/moby/moby/pull/40167 to rename to `statsV1`